### PR TITLE
Relaxed shared dependencies

### PR DESCRIPTION
**Pull request by pasha**

### CODE REVIEW

**Review by michaelstalker — APPROVED** [review]

File: phenotype.gemspec [hunk 1]
```diff
@@ -2,19 +2,19 @@ require './lib/phenotype/version'
...
-  s.add_development_dependency 'pry', '~> 0.10'
-  s.add_dependency 'rack', '~> 1'
-  s.add_dependency 'require_all', '~> 1.3'
-  s.add_development_dependency 'rake', '~> 10.5'
-  s.add_development_dependency 'rspec', '~> 3.4'
+  s.add_dependency 'rack', '>= 1'
+  s.add_dependency 'require_all', '>= 1'
+  s.add_development_dependency 'pry', '>= 0.10'
+  s.add_development_dependency 'rake', '>= 10'
+  s.add_development_dependency 'rspec', '>= 3'
```
Comment (Ebtoulson) [inline phenotype.gemspec]: was the intention to allow older versions than what was previously supported for `require_all`, `rake` and `rspec`?
`~> 3.4` is equivalent to `3.4 >= x < 3.5` but you are now allowing `rspec` older than `3.4`

Comment (pasha) [reply phenotype.gemspec]: good point, will update

Comment (michaelstalker) [reply phenotype.gemspec]: `~> 3.4` is equivalent to `3.4 <= x < 4.0`, but Eric is right that `>= 3` will allow older versions of `rspec`.

Comment (Ebtoulson) [reply phenotype.gemspec]: > ~> 3.4 is equivalent to 3.4 <= x < 4.0

d'oh ur right

Comment (pasha) [reply phenotype.gemspec]: got that fixed, also added travis ci build pipeline

**Review by Ebtoulson — APPROVED** [review]

**Pull request merged**